test/count.test.js

```diff
@@ -0,0 +1,57 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+
+var _ = require('lodash');
+var should = require('should');
+var COUNT_OF_SAMPLES = 70;
+var db, TestCountUser;
+
+function create50Samples() {
```

Comment:
Please rename the function to just `createSamples`. It's misleading right now as it creates 70 samples but is called `create50Samples`

Reply:
good catch